### PR TITLE
[TF] Add handling of non-distributed strategy

### PR DESCRIPTION
**Pull request by andrey-churkin**

### CODE REVIEW

File: beta/examples/tensorflow/common/object_detection/evaluation/coco_evaluator.py [hunk 1]
```diff
@@ -227,10 +227,6 @@ def update(self, predictions, groundtruths=None):
                 groundtruths['height'] = sizes[:, 0]
                 groundtruths['width'] =  sizes[:, 1]
 
-            if 'num_detections' not in groundtruths:
```
Comment (alexsu52) [inline beta/examples/tensorflow/common/object_detection/evaluation/coco_evaluator.py:230]: Could you comment on why we need these changes?

File: beta/examples/tensorflow/segmentation/train.py [hunk 2]
```diff
@@ -45,7 +44,8 @@ def get_argument_parser():
                                         save_checkpoint_freq=False,
                                         export_args=False,
                                         print_freq=False,
-                                        dataset_type=False)
+                                        dataset_type=False,
+                                        cpu_only=False)
```
Comment (alexsu52) [inline beta/examples/tensorflow/segmentation/train.py:48]: Do you figure out the root of the training issue on CPU?

File: beta/examples/tensorflow/common/object_detection/evaluation/coco_evaluator.py (hunk 1 above)
Comment (andrey-churkin) [reply beta/examples/tensorflow/common/object_detection/evaluation/coco_evaluator.py:230]: This code is not necessary now. In this place, `'num_detection'` always is in `groundtruths` and already has the required format.

File: beta/examples/tensorflow/segmentation/train.py (hunk 2 above)
Comment (andrey-churkin) [reply beta/examples/tensorflow/segmentation/train.py:48]: I tried. The problem sometimes is not reproduced. I could try to fix it again.

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**